docs/docs/platforms/OPENSHIFT.md

```diff
@@ -47,6 +53,31 @@ volumes:
       emptyDir: {}
 ```
 
+While it's possible to declare runtime volumes for empty directories, it is not possible for `${APPLICATION_HOME}/conf`. When starting up, Jira and Confluence generate a few configuration files which is a part of image entrypoint. Without anyuid SCC, an unprivileged user can't write to `${APPLICATION_HOME}/conf`. When starting Jira in OpenShift without anyuid SCC attached to jira service account, you will see the following log:
+
+```
+INFO:root:Generating /etc/container_id from template container_id.j2
+WARNING:root:Permission problem writing '/etc/container_id'; skipping
+INFO:root:Generating /opt/atlassian/jira/conf/server.xml from template server.xml.j2
+WARNING:root:Permission problem writing '/opt/atlassian/jira/conf/server.xml'; skipping
+INFO:root:Generating /opt/atlassian/jira/atlassian-jira/WEB-INF/classes/seraph-config.xml from template seraph-config.xml.j2
+WARNING:root:Permission problem writing '/opt/atlassian/jira/atlassian-jira/WEB-INF/classes/seraph-config.xml'; skipping
+INFO:root:Generating /var/atlassian/application-data/jira/dbconfig.xml from template dbconfig.xml.j2
+WARNING:root:Could not chown path /var/atlassian/application-data/jira/dbconfig.xml to jira:jira due to insufficient permissions.
+INFO:root:Running Jira with command '/opt/atlassian/jira/bin/start-jira.sh', arguments ['/opt/atlassian/jira/bin/start-jira.sh', '-fg']
+executing as current user
+```
+
+While this is a non fatal error and Jira is able to proceed with startup, failure to properly generate configuration files like server.xml will result in a number of error when using Jira.
+
+To mitigate the problem, either attach anyuid policy to jira (or confluence) service account **or** build your own container image if existing security practices do not allow anyuid. You need to inherit Jira (or Confluence) official image and make a few directories/files writable for users belonging to a `root` group (which users in OpenShfit containers belong to):
```

Comment:
"need to inherit Jira" -> "need to inherit the Jira"?

Reply:
Fixed